### PR TITLE
Allow editing project description / Game description

### PR DESCRIPTION
**Pull request by ClementPasteau**
This description will be used when published on the web (Liluo)
This will allow the author to add details about their game (story, how to play, etc...)

Screenshot 2022-02-10 at 18 21 16

### CODE REVIEW

File: Core/GDCore/Project/Project.h [hunk 1]
```diff
@@ -65,6 +65,16 @@ class GD_CORE_API Project : public ObjectsContainer {
    */
   const gd::String& GetName() const { return name; }
 
+  /**
+   * \brief Change project description
```
Comment (4ian) [inline Core/GDCore/Project/Project.h:69]: Change the

File: Core/GDCore/Project/Project.h [hunk 2]
```diff
@@ -65,6 +65,16 @@ class GD_CORE_API Project : public ObjectsContainer {
    */
   const gd::String& GetName() const { return name; }
 
+  /**
+   * \brief Change project description
+   */
+  void SetDescription(const gd::String& description_) { description = description_; };
+
+  /**
+   * \brief Get project description
```
Comment (4ian) [inline Core/GDCore/Project/Project.h:74]: Get the project description.

File: newIDE/app/src/GameDashboard/GameDetailsDialog.js [hunk 3]
```diff
@@ -253,12 +254,22 @@ export const GameDetailsDialog = ({
...
                 floatingLabelText={<Trans>Game name</Trans>}
+                floatingLabelFixed={true}
+              />
+              <TextField
+                value={publicGame.description || ''}
+                readOnly
+                fullWidth
+                floatingLabelText={<Trans>Game description</Trans>}
+                floatingLabelFixed={true}
+                hintText={t`No description defined.`}
```
Comment (4ian) [inline newIDE/app/src/GameDashboard/GameDetailsDialog.js:270]: `No description set.`?

**Comment by Bouh** [comment]
We could add a tags input for identity games, such as games made during the jam.

**Pull request merged**